index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Microverse Project by Chris</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="stylesheet" href="assets/css/desktop.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="/">My Logo</a>
+      <button onclick="return;" type="button">
+        <span>
+          <img
+            src="assets/images/icons/Union.png" 
+            alt="nav button">
+        </span>
+      </button>
+    </nav>
+  </header>
+  <main>
+    <div class="backgrounds">
+      <img class="ellipse11"
+        src="assets/images/icons/Ellipse 11.png"
+        alt="background icon">
+      <img class="ellipse8"
+        src="assets/images/icons/Ellipse 8.png"
+        alt="background icon">
+      <img class="ellipse5"
+        src="assets/images/icons/Ellipse 5.png"
+        alt="background icon">
+      <img class="rect7"
+        src="assets/images/icons/Rectangle 7.png"
+        alt="background icon">
+      <img class="ellipse12"
+        src="assets/images/icons/Ellipse 12.png"
+        alt="background icon">
+      <img class="ellipse13"
+        src="assets/images/icons/Ellipse 13.png"
+        alt="background icon">
+      <img class="ellipse14"
+        src="assets/images/icons/Ellipse 14.png"
+        alt="background icon">
+      <img class="ellipse15"
+        src="assets/images/icons/Ellipse 15.png"
+        alt="background icon">
+      <img class="ellipse3"
+        src="assets/images/icons/Ellipse 3.png"
+        alt="background icon">
+      <img class="ellipse4"
+        src="assets/images/icons/Ellipse 4.png"
+        alt="background icon">
+      <img class="rect4"
+        src="assets/images/icons/Rectangle 4.png"
+        alt="background icon">
+      <img class="rect5"
+        src="assets/images/icons/Rectangle 5.png"
+        alt="background icon">
+      <img class="rect6"
+        src="assets/images/icons/Rectangle 6.png"
+        alt="background icon">
+    </div>
+    <div class="content">
+      <h2>
+        I’m Alexandra.
+        Glad to see you!
+      </h2>
```

Comment:
- [ ] This part is the main `title` of this webpage. The main `title` should always be wrapped with `h1` tag for accessibility and semantic reason. Kindly fix this issue.

Reply:
I have changed it to <h1>